WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -72,6 +72,7 @@
 
 int ddLogLevel                                                  = LOG_LEVEL_INFO;
 static NSString * const kUsageTrackingDefaultsKey               = @"usage_tracking_enabled";
+static NSString * const MustShowWhatsNewPopup = @"MustShowWhatsNewPopup";
```

Comment:
The fact this doesn't line up with the other const makes me cry. :crying_cat_face:  (kidding)